### PR TITLE
Allow users to save and unsave places

### PR DESCRIPTION
**Pull request by emmapaczkowski**
Allow user to save places to the Firebase real time data base when they press the star on the place card and visa versa unsave places when they unclick the star.

### CODE REVIEW

File: where-to-travel/src/main/webapp/script.js [hunk 1]
```diff
@@ -332,11 +332,11 @@ function populatePlaces(placeArray) {
 
   $('.icon').click(function() {
     $(this).toggleClass('press');
-    if (firebase.auth().currentUser ) {
+    if (firebase.auth().currentUser && $(this).hasClass('press') == true) {
```
Comment (wjiamin) [inline where-to-travel/src/main/webapp/script.js:335]: I think you can remove the == true part, same with the false statement.

$(this).hasClass('press') == false  => (!$(this).hasClass('press'))

File: where-to-travel/src/main/webapp/script.js [hunk 2]
```diff
@@ -346,6 +346,12 @@ function populatePlaces(placeArray) {
         time: time,
       }
       ref.set(data);
+    } else if (firebase.auth().currentUser && $(this).hasClass('press') == false) {
+      const name = $(this).parent().parent().parent().attr('placeName');
+       // Delete user saved places when the star is not pressed
+      var uID = firebase.auth().currentUser.uid;
```
Comment (wjiamin) [inline where-to-travel/src/main/webapp/script.js:352]: Pull name, uID, and ref out of the if statements since it is shared in both branches?

Comment (andy7809) [reply where-to-travel/src/main/webapp/script.js:352]: +1, name, uID and ref are defined on lines 336, 341, and 342 respectively already

**Review by andy7809 — COMMENTED** [review]
Agree with both of Jasmine suggestions, after that, lgtm

File: where-to-travel/src/main/webapp/script.js (hunk 1 above)
Comment (emmapaczkowski) [reply where-to-travel/src/main/webapp/script.js:335]: done

**Review by saileshsimhadri — APPROVED** [review]

**Pull request merged**